docs/modules/ROOT/pages/dapp.adoc

```diff
@@ -0,0 +1,337 @@
+= Building a Dapp with Solidity Hot Loader
+
+== Introduction
+
+In this guide, we will build a simple Dapp using the Solidity Hot Loader. We will create a Counter Dapp, which displays the value of a Counter that can be increased. We will update the code in the smart contract to show the power of the Solidity Hot Loader to automatically update the Dapp.
```

Comment:
```suggestion
In this guide we will build a simple Dapp using the Solidity Hot Loader: it will display the value of a `Counter` that can be increased. Then, we will update the code in the smart contract to show the power of the Solidity Hot Loader by automatically updating the Dapp.
```